Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ExternalDB/ExternalDBTableFieldForm.tsx

```diff
@@ -1,7 +1,7 @@
 //******************************************************************************************************
 //  ExternalDBTableFieldForm.tsx - Gbtc
 //
-//  Copyright � 2023, Grid Protection Alliance.  All Rights Reserved.
+//  Copyright � 2023, Grid Protection Alliance.  All Rights Reserved.
```

Comment:
```suggestion
//  Copyright © 2023, Grid Protection Alliance.  All Rights Reserved.
```

not sure why this keeps happening. someone must have their settings in VS or VSCode set incorrectly